### DISCUSSION TITLE
Accessor of one col to be accessed by another column

### DISCUSSION THREAD

**Discussion by Varshisri** [TanStack/table#2579] [General]
I have a table displayed and in that I have few columns and in one column I want to add a button and enble it only if my other columns value is so and so.
eg: const columns = [
{
Header: "col 1",
accessor: "columnOne",
},
{
Header: "col 2",
accessor: "coulmnTwo",
},
{
Header: "Edit",
accessor: "editProp",
Cell: () => (
<Button
isDisabled ={} />
},
];
now in this the edit accessor should be shown when only column two data matches "SUBMIT".
I would like to know how can i access the col2 accessor from eidt column.

**Comment by gargroh** [comment]
You will get arguments in Cell...there you can row data by accessor and also original
